packages/default-storage/example/metro.config.js

```diff
@@ -0,0 +1,18 @@
+const path = require("path");
+const { getDefaultConfig } = require("@react-native/metro-config");
+const { getConfig } = require("react-native-builder-bob/metro-config");
+const pkg = require("../package.json");
+
+const root = path.resolve(__dirname, "..");
+
+/**
+ * Metro configuration
+ * https://facebook.github.io/metro/docs/configuration
+ *
+ * @type {import('metro-config').MetroConfig}
+ */
+module.exports = getConfig(getDefaultConfig(__dirname), {
+  root,
+  pkg,
+  project: __dirname,
+});
```

Comment:
We should keep the contents of the previous file:
```suggestion
const { makeMetroConfig } = require("@rnx-kit/metro-config");
const path = require("node:path");

module.exports = makeMetroConfig({
  projectRoot: path.join(__dirname, "example"),
  watchFolders: [__dirname],
  resolver: {
    extraNodeModules: {
      "@react-native-async-storage/async-storage": __dirname,
    },
  },
  transformer: {
    getTransformOptions: async () => ({
      transform: {
        experimentalImportSupport: false,
        inlineRequires: false,
      },
    }),
  },
});
```
This will ensure RN works in future versions as well.